setup.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+
+from setuptools import setup
+
+
+setup(
+    name='dockhub',
+    author='ckolos',
+    version='1.0.0',
+    url='https://github.com/ckolos/dockhub.py/',
+    install_requires=[
+        'requests<3.0.0',
+        'click<8.0.0'
```

Comment:
requests is currently at 2.23.0 and click is at 7.1.2. This is the part where we play poker with the universe....

We are gambling on the fact that click and requests are responsible projects that follow semver and that our stuff won't break as they release patch and minor updates. If they're cool, then we're cool and we don't have to release an update of dockhub just to fix the dependencies. I think that's ok here. I'm a cowboy; on a steel horse I ride. If you feel differently, we should change it.